libs/ui-components/src/components/common/PageNavigation.tsx

```diff
@@ -68,21 +69,24 @@ const PageNavigation = ({ children }: React.PropsWithChildren) => {
   return (
     <>
       <PageSection variant="light" padding={{ default: 'noPadding' }}>
-        <Toolbar isFullHeight isStatic className="fctl-app_toolbar">
-          <ToolbarContent>
-            {hasChildren && <ToolbarItem>{children}</ToolbarItem>}
-            {showOrganizationSelection && (
-              <ToolbarItem>
-                <OrganizationDropdown
-                  organizationName={currentOrgDisplayName}
-                  onSwitchOrganization={() => {
-                    setShowOrganizationModal(true);
-                  }}
-                />
-              </ToolbarItem>
-            )}
-          </ToolbarContent>
-        </Toolbar>
+        <Masthead id="global-actions-masthead">
+          <MastheadContent>
+            <Toolbar isFullHeight isStatic className="fctl-subnav_toolbar">
+              <ToolbarContent>
+                {showOrganizationSelection && (
```

Comment:
Keeping this condition despite being redundant, since it won't be once we add the new "Copy login command" option to the navigation bar.